packages/graphiql-toolkit/src/create-fetcher/create-sse-fetcher.ts

```diff
@@ -0,0 +1,87 @@
+import type {
+  ClientOptions,
+  createClient as createClientType,
+  ExecutionResult,
+} from 'graphql-sse';
+import { Fetcher, FetcherParams } from './types';
+
+/**
+ * Based on https://gist.github.com/enisdenjo/d7bc1a013433502349d2763c3d2f2b79
+ */
+export async function createSseFetcher(opts: ClientOptions): Promise<Fetcher> {
+  const { createClient } =
+    process.env.USE_IMPORT === 'false'
+      ? (require('graphql-sse') as { createClient: typeof createClientType })
+      : await import('graphql-sse');
+
+  const sseClient = createClient({
+    retryAttempts: 0,
+    // @ts-expect-error
+    singleConnection: true, // or use false if you have an HTTP/2 server
```

Comment:
How is this supposed to be configured by the user?

Reply:
Nice catch, we need to pass options in
https://github.com/graphql/graphiql/blob/2936a79bf3d8d2a7723987bdd6d268d3a627293e/packages/graphiql-toolkit/src/create-fetcher/createFetcher.ts#L50-L52